src/net/sf/memoranda/ui/EventDialog.java

```diff
@@ -546,7 +547,14 @@ public void noRepeatRB_actionPerformed(ActionEvent e) {
     }
 
     void okB_actionPerformed(ActionEvent e) {
-        this.dispose();
+    	if (textField.getText().isEmpty())
+    	{
+    		JOptionPane.showMessageDialog(eventPanel, "Detail cannot be empty.");
```

Comment:
I thought we were doing { at the end of a line and } on its own line per the CG?